-------------------------------------------------------------------------------
Test set: TestSuite
-------------------------------------------------------------------------------
Tests run: 2, Failures: 1, Errors: 0, Skipped: 1, Time elapsed: 4.942 s <<< FAILURE! - in TestSuite
loginTrue(com.course.cases.loginTest)  Time elapsed: 2.771 s  <<< FAILURE!
org.apache.http.conn.HttpHostConnectException: Connect to localhost:8888 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused: connect
	at com.course.cases.loginTest.getResult(loginTest.java:72)
	at com.course.cases.loginTest.loginTrue(loginTest.java:49)
Caused by: java.net.ConnectException: Connection refused: connect
	at com.course.cases.loginTest.getResult(loginTest.java:72)
	at com.course.cases.loginTest.loginTrue(loginTest.java:49)

